packages/smb-onboarder/src/components/ui/negotiators-list/__mocks__/negotiators.ts

```diff
@@ -1,5 +1,6 @@
 import { NegotiatorsQueryResponse, NegotiatorUpdateParams, NegotiatorCreateParams } from '../negotiators-list'
-import { NegotiatorModel } from '@reapit/foundations-ts-definitions/types'
+import { NegotiatorModel } from '@reapit/elements/node_modules/@reapit/foundations-ts-definitions/types'
```

Comment:
Why do we need to import like this?

Reply:
In the original commit (`08884de6a8d70bbd9ccfed34b36ce026121fc881`). I saw it was written like this... Seems like other commit has addressed this issue